velox/docs/functions.rst

```diff
@@ -62,91 +62,99 @@ for :doc:`all <functions/presto/coverage>` and :doc:`most used
-    :func:`date`                            :func:`map_normalize`                   :func:`typeof`
-    :func:`date_add`                        :func:`map_subset`                      :func:`upper`
-    :func:`date_diff`                       :func:`map_top_n`                       :func:`url_decode`
-    :func:`date_format`                     :func:`map_values`                      :func:`url_encode`
-    :func:`date_parse`                      :func:`map_zip_with`                    :func:`url_extract_fragment`
-    :func:`date_trunc`                      :func:`md5`                             :func:`url_extract_host`
-    :func:`day`                             :func:`millisecond`                     :func:`url_extract_parameter`
-    :func:`day_of_month`                    :func:`minus`                           :func:`url_extract_path`
-    :func:`day_of_week`                     :func:`minute`                          :func:`url_extract_port`
-    :func:`day_of_year`                     :func:`mod`                             :func:`url_extract_protocol`
-    :func:`degrees`                         :func:`month`                           :func:`url_extract_query`
-    :func:`distinct_from`                   :func:`multimap_from_entries`           :func:`week`
-    :func:`divide`                          :func:`multiply`                        :func:`week_of_year`
-    :func:`dow`                             :func:`nan`                             :func:`weibull_cdf`
-    :func:`doy`                             :func:`negate`                          :func:`width_bucket`
-    :func:`e`                               :func:`neq`                             :func:`wilson_interval_lower`
-    :func:`element_at`                      :func:`ngrams`                          :func:`wilson_interval_upper`
-    :func:`empty_approx_set`                :func:`no_keys_match`                   :func:`xxhash64`
-    :func:`ends_with`                       :func:`no_values_match`                 :func:`year`
-    :func:`eq`                              :func:`none_match`                      :func:`year_of_week`
-    :func:`exp`                             :func:`normal_cdf`                      :func:`yow`
-    :func:`f_cdf`                           not                                     :func:`zip`
-    :func:`filter`                          :func:`parse_datetime`                  :func:`zip_with`
-    :func:`find_first`                      :func:`pi`
-    ======================================  ======================================  ======================================  ==  ======================================  ==  ======================================
+    ========================================  ========================================  ========================================  ==  ========================================  ==  ========================================
```

Comment:
@pramodsatya : Can you generate the doc and check the end result. The viewer is showing a strange part with an entry for not
![image](https://github.com/user-attachments/assets/e1448446-0098-4f31-9119-31905b68d57e)

and another place with function 'in' as well

![image](https://github.com/user-attachments/assets/affd82df-db88-4887-938f-a02036fb630b)

Reply:
Thanks for pointing it out @aditi-pandit, the links for `not` and `in` appear to be missing in the current prestosql coverage map doc as well: https://facebookincubator.github.io/velox/functions.html .

Reply:
@pramodsatya : Lets debug why they are not present. But the changes here are sufficient for this PR.